<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tab_RL_Calc">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>容抗 感抗</string>
     </attribute>
     <widget class="QComboBox" name="InductorComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="InductanceLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Inductance</string>
      </property>
     </widget>
     <widget class="QLabel" name="FrequencyInductanceLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
     <widget class="QComboBox" name="InductorFrequencyComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextBrowser" name="InductanceOutput">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="InductorCalcpushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="InductorInductanceEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="InductorFrequencyEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="InductanceLabel_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ω</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CapcitanceComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="CapcitanceFrequencyComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="CapcitorCapcitanceEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="CapcitanceCalcpushButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>340</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calc</string>
      </property>
     </widget>
     <widget class="QLabel" name="InductanceLabel_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>270</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ω</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CapcitanceOutput">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
     <widget class="QLabel" name="InductanceLabel_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CapcCapcitance</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CapcitorFrequencyEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="FrequencyInductanceLabel_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Lc,Rc,LR转折频率</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>页</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
